src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/planner/QueryPlannerTest.java

```diff
@@ -110,6 +114,12 @@ public static void initLogger() {
     public void init() {
         MockitoAnnotations.initMocks(this);
 
+        SqlSettings settings = spy(new SqlSettings());
+        // Force return empty list to avoid ClusterSettings be invoked which is a final class and hard to mock.
+        // In this case, default value in Setting will be returned all the time.
+        doReturn(emptyList()).when(settings).getSettings();
```

Comment:
Why are we stubbing `getSettings()` method - is it being used anywhere?

Reply:
It's used in `setSettings()` here. https://github.com/opendistro-for-elasticsearch/sql/blob/7c57d72ad22b1df39b5f89838a41fbf2e8a76829/src/main/java/com/amazon/opendistroforelasticsearch/sql/esdomain/LocalClusterState.java#L125